### ISSUE TITLE
app doesnt open

### ISSUE THREAD

**Issue by iamLazyCode** [Juby210/SwiftBackupPrem#4]
hi i have moddified 3 times as per video but swift backup just doesn't open it just closes if I active swift backup in lsposed

**Comment by five-eighteen** [comment]
I just went through the setup process today and it's working fine with Swift Backup v5.0.4. With it deactivated in lsposed try first scrolling to the bottom of Swift Backup and pressing Restart.

**Comment by iamLazyCode** [comment]
> I just went through the setup process today and it's working fine with Swift Backup v5.0.4. With it deactivated in lsposed try first scrolling to the bottom of Swift Backup and pressing Restart.

do I need to log out of the account if I was using on swift backup or can I use the same account ? or do I need new account ? and also can you link to the lsposed your using >

**Comment by five-eighteen** [comment]
> do I need to log out of the account if I was using on swift backup or can I use the same account ? or do I need new account ? and also can you link to the lsposed your using

I believe hitting Restart in Swift Backup logs you out, but you could also just clear the cache and storage of the app.

You should use whichever account you created the project with for Google OAuth and Firebase (in the video). I'm actually not sure where I got this lsposed from but it says version 14 in the app settings.

**Comment by iamLazyCode** [comment]
> > do I need to log out of the account if I was using on swift backup or can I use the same account ? or do I need new account ? and also can you link to the lsposed your using
> 
> I believe hitting Restart in Swift Backup logs you out, but you could also just clear the cache and storage of the app.
> 
> You should use whichever account you created the project with for Google OAuth and Firebase (in the video). I'm actually not sure where I got this lsposed from but it says version 14 in the app settings.

great il try again with this...also only lsposed I find is this

https://github.com/LSPosed/LSPosed please confirm

**Comment by five-eighteen** [comment]
> > > do I need to log out of the account if I was using on swift backup or can I use the same account ? or do I need new account ? and also can you link to the lsposed your using
> > 
> > 
> > I believe hitting Restart in Swift Backup logs you out, but you could also just clear the cache and storage of the app.
> > You should use whichever account you created the project with for Google OAuth and Firebase (in the video). I'm actually not sure where I got this lsposed from but it says version 14 in the app settings.
> 
> great il try again with this...also only lsposed I find is this
> 
> https://github.com/LSPosed/LSPosed please confirm

Yes, that should be it

**Comment by iamLazyCode** [comment]
> > > > do I need to log out of the account if I was using on swift backup or can I use the same account ? or do I need new account ? and also can you link to the lsposed your using
> > > 
> > > 
> > > I believe hitting Restart in Swift Backup logs you out, but you could also just clear the cache and storage of the app.
> > > You should use whichever account you created the project with for Google OAuth and Firebase (in the video). I'm actually not sure where I got this lsposed from but it says version 14 in the app settings.
> > 
> > 
> > great il try again with this...also only lsposed I find is this
> > https://github.com/LSPosed/LSPosed please confirm
> 
> Yes, that should be it

thanks a lot mate let me try again and report back

**Comment by iamLazyCode** [comment]
> > > > do I need to log out of the account if I was using on swift backup or can I use the same account ? or do I need new account ? and also can you link to the lsposed your using
> > > 
> > > 
> > > I believe hitting Restart in Swift Backup logs you out, but you could also just clear the cache and storage of the app.
> > > You should use whichever account you created the project with for Google OAuth and Firebase (in the video). I'm actually not sure where I got this lsposed from but it says version 14 in the app settings.
> > 
> > great il try again with this...also only lsposed I find is this
> > 
> > https://github.com/LSPosed/LSPosed please confirm
> 
> Yes, that should be it

Still same if u have time i can record screen can u please check what i am doing wrong...would be great help

**Comment by iamLazyCode** [comment]
> > > > do I need to log out of the account if I was using on swift backup or can I use the same account ? or do I need new account ? and also can you link to the lsposed your using
> > > 
> > > 
> > > I believe hitting Restart in Swift Backup logs you out, but you could also just clear the cache and storage of the app.
> > > You should use whichever account you created the project with for Google OAuth and Firebase (in the video). I'm actually not sure where I got this lsposed from but it says version 14 in the app settings.
> > 
> > 
> > great il try again with this...also only lsposed I find is this
> > https://github.com/LSPosed/LSPosed please confirm
> 
> Yes, that should be it

please reply what else can i do

**Comment by Veritera** [comment]
App says "Enable module in LSPOSED manager before using it" while the module is already acticated.

**Comment by iamLazyCode** [comment]
> App says "Enable module in LSPOSED manager before using it" while the module is already acticated.

I have activated it sir...isnit working fine for u on latest version ?

**Comment by NVG-064** [comment]
If you found "Enable module in LSPOSED manager before using it" even though you're already activated, it's your LSPosed fork issue, not the module itself. Please change to another LSPosed fork version. Take an example fork: LSPosed npm fork

**Comment by iamLazyCode** [comment]
> If you found "Enable module in LSPOSED manager before using it" even though you're already activated, it's your LSPosed fork issue, not the module itself. Please change to another LSPosed fork version. Take an example fork: LSPosed npm fork

il try again but could u link to exact repo please...thanks

**Comment by NVG-064** [comment]
https://t.me/rormzhstjxm/100

No repo available, but you can check source code inside the channel

**Comment by iamLazyCode** [comment]
> https://t.me/rormzhstjxm/100
> 
> No repo available, but you can check source code inside the channel

Thankyou so much il check now
